src/org/java/training/BlurryRotator.java

```diff
@@ -0,0 +1,73 @@
+package org.java.training;
+
+import javafx.application.Application;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.effect.BoxBlur;
+import javafx.scene.effect.Reflection;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.transform.Rotate;
+import javafx.scene.transform.Scale;
+import javafx.stage.Stage;
+
+public class BlurryRotator extends Application {
+
+	double angle = 0.0, scaleFactor = 0.4, blurAmount = 1.0;
+	Reflection reflect = new Reflection();
+	Rotate rotate = new Rotate();
+	Scale scale = new Scale(scaleFactor, scaleFactor);
+	BoxBlur
+		blur = new BoxBlur(1.0, 1.0, 1),
+		rotateBlur = new BoxBlur(5.d, 5.d, 2);
+	
```

Comment:
Yeah, I like to group instantiation like this. It visually separates groups of related instances quite nicely.